app/scripts/directives/overview/mobileClientRow.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+
+(function () {
+  angular.module('openshiftConsole').component('mobileClientRow', {
+    controller: [
+      '$scope',
+      '$filter',
+      '$routeParams',
+      'ProjectsService',
+      'DataService',
+      'APIService',
+      'MobileClientsService',
+      'ListRowUtils',
+      'Navigate',
+      'AuthorizationService',
+      MobileAppRow,
+    ],
+    controllerAs: 'row',
+    bindings: {
+      apiObject: '<',
+      state: '<',
+      bindings: '<'
+    },
+    templateUrl: 'views/overview/_mobile-client-row.html'
+  });
+
+  function MobileAppRow($scope, $filter, $routeParams, ProjectsService, DataService, APIService, MobileClientsService, ListRowUtils, Navigate, AuthorizationService) {
+    var row = this;
+    row.installType = '';
+
+    _.extend(row, ListRowUtils.ui);
+
+    row.bundleDisplay = "bundle id: " + row.apiObject.spec.appIdentifier;
```

Comment:
Bundle display and install type might make good files.

If you keep them in the controller, you'd probably want to put them in `$onChanges`